src/instances/views/index.ecr

```diff
@@ -88,7 +88,7 @@
   <body>
     <div class="header">
       <p>
-        <span>Open an issue or a pull request to add your instance on <a href="https://github.com/iv-org/documentation/">the documentation repository</a>.</span>
+        <span>Open an issue to add your instance on <a href="https://github.com/iv-org/documentation/">the documentation repository</a>.</span>
```

Comment:
```suggestion
        <span>To add your instance please open an issue on <a href="https://github.com/iv-org/documentation/">the documentation repository</a>.</span>
```

Imo this entire sentence should probably be rephrased to clarify that to add an instance to the list below one should open an issue on the documentation repo.

Reply:
I second that suggestion